assets/src/dashboard/app/views/templates-gallery/index.js

```diff
@@ -19,15 +19,54 @@
  */
 import { __ } from '@wordpress/i18n';
 
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
 /**
  * Internal dependencies
  */
-import { ViewHeader } from '../../../components';
+import { Dropdown, ViewHeader, DropdownContainer } from '../../../components';
+import { DROPDOWN_TYPES } from '../../../constants';
+
+const ExploreFiltersContainer = styled.div`
+  padding: 0 20px 20px;
+  border-bottom: 1px solid #eee;
```

Comment:
`${({theme}) => 1px solid ${theme.colors.gray50}};`

should this just be in the theme as 1 string? i feel like I'm seeing this border pattern alot.